R/resequence.R

```diff
@@ -0,0 +1,83 @@
+#' Resorting and filtering dates
+#'
+#' Resorting and filtering dates
+#' @param data a dataframe
+#' @param vars a character vector identifying columns
+#' in the dataframe to sequence
+#' @param unity a string identifying how multiple
+#' entries may be glued together.
+#' By default, ".." are used in accordance with
+#' ISO 8601-2_2019(E) standards.
+#' @return a dataframe/columns
+#' @import lubridate
+#' @importFrom stats na.omit
+#' @examples
+#' data <- data.frame(Sign = c("2000-01-01", "2001-01-01",
+#' "2001-01-01..2000-01-01", "2000-01-01", NA, "2016-12-31~"),
+#' Force = c("2001-01-01", "2000-01-01",
+#' "2001-01-01", NA, "2001-XX-XX", "~1816_01_01"))
+#' resequence(data, c("Sign", "Force"))
+#' @export
+resequence <- function(data, vars, unity = "\\.\\.") {
+
```

Comment:
I wonder whether `resequence()` makes more sense operating on vectors? Also, it could be an S3 method, and could work on either `messydt` class or e.g. `Date` or `POSIXct`?

Reply:
@jhollway @henriquesposito can I add this to issue #26 and leave it out from this PR?

Reply:
Fine by me.

Reply:
Yes, component sounds more clear/consistent. Thank you.